src/plugins/op-upload-resource-adapter.js

```diff
@@ -7,16 +9,18 @@ export default class OpUploadResourceAdapter {
 
     upload() {
 		const resource = this.resource;
-        if (!(resource && resource.uploadAttachments)) {
-			const resourceContext = resource ? resource.name : 'Missing context';
-            console.warn(`uploadAttachments not present on context: ${resourceContext}`);
-            return Promise.reject("You're not allowed to upload attachments on this resource.");
+		const resourceService = getOPService(this.editor, 'attachmentsResourceService');
```

Comment:
yes it will, but this will be tested in the core so it should fail there. A console error (or similar) raised here is not any better than a hard error imho